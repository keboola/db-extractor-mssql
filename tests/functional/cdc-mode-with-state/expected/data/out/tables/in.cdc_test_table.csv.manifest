{
    "destination": "in.cdc_test_table",
    "incremental": false,
    "primary_key": [
        "_Weir%d I-D"
    ],
    "columns": [
        "_Weir%d I-D",
        "Weir%d Na-me",
        "someInteger",
        "someDecimal",
        "type",
        "smalldatetime",
        "datetime",
        "KBC__DELETED"
    ]
}