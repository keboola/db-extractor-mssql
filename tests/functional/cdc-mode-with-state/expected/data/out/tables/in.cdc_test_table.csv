"6","yoshi","6","6.60","horse?","2012-01-10 10:25:00","2021-01-05 13:43:27.123","0"
"7","nedData","6","6.60","horse?","2023-08-10 10:25:00","2023-08-10 13:43:27.123","0"
"8","asd","6","6.60","horse?","2023-08-10 10:25:00","2023-08-10 13:43:27.123","0"
